spec/controllers/shipping_api_controller_spec.rb

```diff
@@ -24,13 +26,39 @@
       end
 
       describe "the returned json object" do
-        before :each do
-          @object = JSON.parse(response.body)
+        # before :each do
+        #   VCR.use_cassette("/rates", record: :new_episodes) do
+        #     post :rates, params
+        #   end
+        # end
+        let(:object) { JSON.parse(response.body) }
+
+        it "is an array of hashes" do
+          expect(object).to be_an_instance_of Array
+          expect(object.first).to be_an_instance_of Hash
+        end
+
+        it "includes only service, price, and delivery_date" do
+          object.each do |instance|
+            expect(instance.keys).to eq(["service", "price", "delivery_date"])
+          end
```

Comment:
Would you be willing to add another line to this one that checks to make sure that these are the ONLY keys included?